### ISSUE TITLE
Still an issue with this call

### ISSUE THREAD

**Issue by felixdkatt** [stripe/stripe-ios#240]
STPAPIClient.sharedClient().createTokenWithPayment(payment, completion: { (token, error) -> Void in

is still giving me an error. 

PaymentController.swift:209:75: Invalid conversion from throwing function of type '(_, _) throws -> Void' to non-throwing function type 'STPCompletionBlock' (aka '(Optional<STPToken>, Optional<NSError>) -> ()')

Is there a Swift 2.0 example on how to create a payment?

The error happens directly on the stripe method call not on the rest of my code.

See gist here:

https://gist.github.com/felixdkatt/0ce3f01643541e6bc884

**Comment by benzguo** [comment]
Hi @felixdkatt – I haven't been able to reproduce this error. Would you mind sending us an example app? Feel free to email me ( bg@stripe.com ).

**Comment by felixdkatt** [comment]
I have a gist of the function I’m using and the responses I’m getting


https://gist.github.com/felixdkatt/0ce3f01643541e6bc884

**Comment by benzguo** [comment]
Yup, I saw the gist, but haven't been able to reproduce the error on my end. An example app demonstrating the compiler error would be very helpful. 

FWIW, we're essentially doing the same thing in the Swift example app: https://github.com/stripe/stripe-ios/blob/master/Example/Stripe%20iOS%20Example%20(Simple)/ViewController.swift#L54-L73

**Comment by felixdkatt** [comment]
i tried using same method and i get the same issue but i ran your example version and it was fine. Not sure what I’m doing different. Still checking...

**Comment by felixdkatt** [comment]
ok looks like Xcode decided to place the blame on the method call and not on the real issue

i had a 
try NSJSONSerialization.dataWithJSONObject(body, options: NSJSONWritingOptions())

where before i didn’t need a try in swift 1.2

to create a NSURL Request

and it needed a do{  }catch{} with it 

Instead of Xcode just telling me that in plain english it highlighted your method call and blamed it on you.

It compiles fine now thanks for the help

**Comment by benzguo** [comment]
:tada:
